lib/shared/log/tasks.js

```diff
@@ -13,7 +13,23 @@ function isObject(val) {
   return typeof val === 'object' && !Array.isArray(val);
 }
 
+function wrapGetTask(getTask) {
+  return function(name) {
+    return createTask(getTask(name));
+  };
+}
+
+function createTask(task) {
+  var unwrapped = task.unwrap ? task.unwrap() : {};
```

Comment:
I think `typeof task.description === 'string'` is best because we don't handle any other cases.